rules/windows/process_creation/win_apt_chafer_mar18.yml

```diff
@@ -67,13 +67,14 @@ logsource:
     product: windows
 detection:
     selection_process1:
-        CommandLine: 
-            - '*\Service.exe i'
-            - '*\Service.exe u'
-            - '*\microsoft\Taskbar\autoit3.exe'
-            - 'C:\wsc.exe*'
+        CommandLine|endswith: 
+            - '\Service.exe i'
+            - '\Service.exe u'
+            - '\microsoft\Taskbar\autoit3.exe'
+        CommandLine|startswith:
+            - 'C:\wsc.exe'
     selection_process2:
-        Image: '*\Windows\Temp\DB\\*.exe'
+        Image|endswith: '\Windows\Temp\DB\\*.exe'
     selection_process3:
-        CommandLine: '*\nslookup.exe -q=TXT*'
-        ParentImage: '*\Autoit*'
+        CommandLine|contains: '\nslookup.exe -q=TXT'
```

Comment:
I would rather split it to:

1. exclude the space
2. implement detection of `nslookup` in the ProcessName field, that's cheaper

Reply:
fixed